battleground-state-changes.html

```diff
@@ -50,11 +50,6 @@
         border-left: none;
     }
 
-    .has-tip:after{
-        content: "\2139\fe0f";
-        margin-left: 5px;
-    }
-
```

Comment:
You'll want to make these changes to battleground-state-changes.html.tmpl instead of battleground-state-changes.html (which is automatically generated by the CI tool when it pulls updates to the JSON file)